doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -1,115 +1,128 @@
 .. _rados-troubleshooting-mon:
 
-=================================
+==========================
  Troubleshooting Monitors
-=================================
+==========================
 
 .. index:: monitor, high availability
 
-When a cluster encounters monitor-related troubles there's a tendency to
-panic, and sometimes with good reason. Losing one or more monitors doesn't
-necessarily mean that your cluster is down, so long as a majority are up,
-running, and form a quorum.
-Regardless of how bad the situation is, the first thing you should do is to
-calm down, take a breath, and step through the below troubleshooting steps.
+If a cluster encounters monitor-related problems, this does not necessarily
+mean that the cluster is in danger of going down. Even if multiple monitors are
+lost, the cluster can still be up and running, as long as there are enough
+surviving monitors to form a quorum.
 
+However serious your cluster's monitor-related problems might be, we recommend
+that you take the following troubleshooting steps.
 
-Initial Troubleshooting
-========================
 
+Initial Troubleshooting
+=======================
 
 **Are the monitors running?**
 
-  First of all, we need to make sure the monitor (*mon*) daemon processes
-  (``ceph-mon``) are running.  You would be amazed by how often Ceph admins
-  forget to start the mons, or to restart them after an upgrade. There's no
-  shame, but try to not lose a couple of hours looking for a deeper problem.
-  When running Kraken or later releases also ensure that the manager
-  daemons (``ceph-mgr``) are running, usually alongside each ``ceph-mon``.
- 
+  First, make sure that the monitor (*mon*) daemon processes (``ceph-mon``) are
+  running. Sometimes Ceph admins either forget to start the mons or forget to
+  restart the mons after an upgrade. Checking for this simple oversight can
+  save hours of painstaking troubleshooting. It is also important to make sure
+  that the manager daemons (``ceph-mgr``) are running. Remember that typical
+  cluster configurations provide one ``ceph-mgr`` for each ``ceph-mon``.
+
+**Can you reach the monitor nodes?**
+
+  In certain rare cases, there are ``iptables`` rules that block access to
+  monitor nodes or TCP ports. These rules might be left over from earlier
+  stress testing or rule development. To check for the presence of such rules,
+  SSH into the server and then try to connect to the monitor's ports
+  (``tcp/3300`` and ``tcp/6789``) using ``telnet``, ``nc``, or a similar tool.
 
-**Are you able to reach to the mon nodes?**
+**Does the ``ceph -s`` command run and receive a reply from the cluster?**
```

Comment:
Here and below we should probably be formal and suggest ``ceph status`` (same command)

Reply:
I've changed all instances of "ceph -s" to "ceph status".